README.md

```diff
@@ -18,33 +20,48 @@ git clone https://github.com/RioChndr/jaksel-language.git
 
 ### Run
 
-If you install from npm, run jaksel script using `jaksel` command
+If you install from npm, run jaksel script using `jaksel -r <filename...>` command
+
+```bash
+jaksel -r example/example1.jaksel
```

Comment:
My suggestion is make this command use old style jaksel <file name>

Reply:
`jaksel <file name>`

Reply:
I added the -r flag for run to make it look the same as the -c flag, if the command runs only jaksel hmm it seems less consistent

Btw thanks for suggestions